maps/src/main/java/org/odk/collect/maps/layers/OfflineMapLayersPicker.kt

```diff
@@ -0,0 +1,84 @@
+package org.odk.collect.maps.layers
+
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.odk.collect.androidshared.ui.GroupClickListener.addOnClickListener
+import org.odk.collect.async.Scheduler
+import org.odk.collect.maps.databinding.OfflineMapLayersPickerBinding
+import org.odk.collect.settings.SettingsProvider
+import org.odk.collect.webpage.ExternalWebPageHelper
+
+class OfflineMapLayersPicker(
```

Comment:
Let's keep this in mind but do not change now. I'm not against but I also think it's not something we definitely need to change.
When it comes to testing I already verify the order of displayed layers in `OfflineMapLayersPickerTest`.
Even if after merging this pr and creating a new one I change my mind and start thinking that yeah we should have it, it would be better to do that later so that I can cherry-pick the commits I've removed easier.